package.json

```diff
@@ -291,7 +290,6 @@
     "glob": "^10.2.2",
     "html-webpack-plugin": "5.5.1",
     "http-proxy": "^1.16.2",
-    "husky": "^8.0.3",
```

Comment:
Could you also remove `lint-staged` and regenerate the `yarn.lock` file by running `yarn`?